lib/ubus.c

```diff
@@ -2438,10 +2452,25 @@ uc_ubus_channel_connect(uc_vm_t *vm, size_t nargs)
 }
 
 
+static uc_value_t *
+uc_ubus_exception_handler_set(uc_vm_t *vm, size_t nargs)
+{
+	uc_value_t *arg = uc_fn_arg(0);
+
+	if (arg && !ucv_is_callable(arg))
```

Comment:
`ucv_is_callable()` is NULL-safe.

Reply:
That wasn't a safety check. The intention is to allow setting the exception handler to null, but reject non-callable non-null values.